linera-service/src/cli_wrappers/wallet.rs

```diff
@@ -1325,7 +1325,7 @@ impl<A> ApplicationWrapper<A> {
         self.raw_query(&format!("query {{ {query} }}")).await
     }
 
-    pub async fn query_json<T: DeserializeOwned>(&self, query: impl AsRef<str>) -> Result<T> {
+    pub async fn fetch_json<T: DeserializeOwned>(&self, query: impl AsRef<str>) -> Result<T> {
```

Comment:
sure

Reply:
Ah, sorry, that was find-and-replace, because I had called the API method `query_json` at first, too. :facepalm: